cdc/log.cc

```diff
@@ -905,24 +947,12 @@ class transformer final : public change_processor {
     void generate_image(operation op, const clustering_key* ck, const one_kind_column_set* affected_columns) {
         assert(op == operation::pre_image || op == operation::post_image);
 
-        auto& res = current_mutation();
-
-        auto image_ck = allocate_new_log_row();
-        set_operation(image_ck, op);
+        assert(_builder);
+        auto image_ck = _builder->allocate_new_log_row();
+        _builder->set_operation(image_ck, op);
```

Comment:
Maybe some overloads in builder for `allocate_new_log_row(op)` and similar?

Reply:
Done